src/content_script.js

```diff
@@ -268,6 +270,14 @@ function shouldBadgeBeSmall(ratioCheck, itemHeight) {
   return false;
 }
 
+function openLoginPrompt(parent, el, htmlBadgeDiv) {
```

Comment:
This change was required with the `isTrusted` checks. I moved this function outside of the `addEventListener` in-line event so that other functions could call it.